src/index.js

```diff
@@ -92,7 +90,7 @@ function syncReduxAndRouter(history, store, selectRouterState = SELECT_STATE) {
 
   const unsubscribeHistory = history.listen(location => {
     const route = {
-      path: history.createPath(location),
+      path: history.createHref(location),
```

Comment:
i tried `createPath` but devtools tests were still failing... did you update deps?